api/app/signals/apps/email_integrations/tests/test_actions.py

```diff
@@ -71,6 +71,12 @@ def setUp(self):
                                      body='{{ text }} {{ created_at }} {{ reaction_request_answer }} '
                                           '{{ ORGANIZATION_NAME }}')
 
+        EmailTemplate.objects.create(key=EmailTemplate.SIGNAL_STATUS_CHANGED_AFGEHANDELD_KTO_NEGATIVE_CONTACT,
+                                     title='Uw melding {{ signal_id }}'
+                                           f' {EmailTemplate.SIGNAL_STATUS_CHANGED_AFGEHANDELD_KTO_NEGATIVE_CONTACT}',
+                                     body='{{ text }} {{ created_at }} {{ reaction_request_answer }} '
+                                          '{{ ORGANIZATION_NAME }}')
+
```

Comment:
Add a `TestSignalHandledNegativeAction` see `TestSignalHandledAction` for an example